spec/runners/unified/assertions.rb

```diff
@@ -156,7 +156,7 @@ def assert_events
         else
           false
         end
-        actual_events = subscriber.wanted_events(@observe_sensitive)
+        actual_events = subscriber.wanted_events(@observe_sensitive[client])
```

Comment:
```suggestion
        actual_events = subscriber.wanted_events(@observe_sensitive.fetch(client))
```

Reply:
I would also index with `client_id` rather than `client`.

Reply:
Or can `nil` be passed to `wanted_events`?

Reply:
nil can (and should) be passed to `wanted_events`...

I indexed by client because that's what we do for the `subscribers` hash but I can change it.

Reply:
In that case it's fine the way it is.